src/templates/global/layout.html.tt

```diff
@@ -21,14 +21,27 @@
 [% END %]
 [% IF layout %]
         <div id="content"[% IF current_route +%] class="[% current_route %]-content"[% END %]>
+            [% IF layout == 'full_width' %]
+                <div class="container">
```

Comment:
Having more than one `.container` in `#content` breaks generic error message [here](https://github.com/binary-com/binary-static/blob/master/src/javascript/binary/base/binary_loader.js#L111)
i.e. browse a real only page using a virtual account (limits, cashier password, ...)
You can also handle this by updating the selector in `displayMessage()`.

Reply:
I see, I'll remove the `.container` then and assign the css values directly to the `ico-banner` class.

Reply:
Thanks.